dotnet/src/SemanticKernel.Skills/Skills.OpenAPI/OpenApi/OpenApiDocumentParser.cs

```diff
@@ -118,6 +129,25 @@ private static IList<RestApiOperationParameter> CreateRestApiOperationParameters
         return result;
     }
 
+    /// <summary>
+    /// Creates REST API operation headers.
+    /// </summary>
+    /// <param name="parameters">The OpenApi parameters</param>
+    /// <returns>The headers.</returns>
+    private static IDictionary<string, string> CreateRestApiOperationHeaders(IList<OpenApiParameter> parameters)
+    {
+        var headers = new Dictionary<string, string>();
```

Comment:
Can we simplify this method? Not sure about `.ToDictionary` performance though.
```
return parameters
    .Where(p => p.In == ParameterLocation.Header)
    .ToDictionary(p => p.Name, p => string.Empty);
```

Reply:
Simplified :)